### ISSUE TITLE
Changes suggestion for the Serialization example

### ISSUE THREAD

**Issue by jimver04** [microsoft/glTF-SDK#110]
In the Serialization example main.cpp

perhaps it worths changing 

std::vector<uint16_t> indices;       
to        
std::vector<uint32_t> indices;

and 

accessorIdIndices = bufferBuilder.AddAccessor(indices, { Microsoft::glTF::TYPE_SCALAR, 								Microsoft::glTF::COMPONENT_UNSIGNED_SHORT}).id;    

to

accessorIdIndices = bufferBuilder.AddAccessor(indices, { Microsoft::glTF::TYPE_SCALAR, 								Microsoft::glTF::COMPONENT_UNSIGNED_INT}).id;

because it took me days to figure out why it was not working with my data.  Some meshes had more than 2^16 vertices and were scrumbled whereas some other meshes were ok. 

Best,
Dimitrios

**Comment by bghgary** [comment]
I'm not sure I understand. Why does it make more sense to use a larger index type to store the indices when the smaller type works? Maybe a comment would suffice?

**Comment by jimver04** [comment]
Yes a comment would be nice there to point out that these should be changed according to the size of data, i.e. when the number of vertices > 65536

**Comment by bghgary** [comment]
Please open a PR. I will review and merge.

**Comment by bghgary** [comment]
Is this still needed?
